### DISCUSSION TITLE
Nach Update Home Assistant, Homematic Geräte nicht mehr verfügbar

### DISCUSSION THREAD

**Discussion by bratbecker** [OpenCCU/OpenCCU#2357] [🇩🇪 Fragen&Antworten] [answered]
Hi zusammen,
nach Update der HA auf Core 2023.7.1, sind die Homematic Geräte nicht mehr ansprechbar.
Aus der Rasperrimatic heraus funktioniert es einwandfrei.
Jemand ähnliches Problem?

**Comment by jens-maus** [comment, marked as answer]
Das kann ich selbst so nicht bestätigen. Ich nutze selbst auch 2023.7.1 und die Kommunikation mit den Homematic/homematicIP Geräten funktioniert tadellos via der "Homematic(IP) Local" Integration. Anleitung dazu siehe hier:

https://github.com/jens-maus/RaspberryMatic/wiki/HomeAssistant-Integration

**Reply by colognebroker** [reply]
Doch es funktionieren definitiv nicht alle geräte einige switches sind nicht verfügbar

**Reply by colognebroker** [reply]
nach einem rollback funktionieren sie wieder

**Reply by jens-maus** [reply]
Beispiele benennen. Hier konnte ich noch keinerlei Probleme feststellen wie gesagt.

**Reply by colognebroker** [reply]
was musst du wissen ?

**Reply by colognebroker** [reply]
HM-LC-Sw1-Pl 
der funktioniert nicht !
Habe drei davon alle gehen nicht nach update

**Reply by colognebroker** [reply]
HM-LC-Sw1PBU-FM
und dieses Gerät funktioniert nicht

**Reply by jens-maus** [reply]
Auch das kann ich leider nicht bestätigen. Ich hab zwar exakt diese Typen nicht, aber z.B. die `HM-LC-Sw1-FM` die ich verbaut habe lassen sich problemlos von HA aus steuern und schalten dann entsprechend.

Aber selbst wenn es ein Problem gibt, dann ist dieses Forum bzw. dieses Projekt hier das falsche um das zu melden. Das gehört hier bei der Homematic Integration gemeldet:

https://github.com/danielperna84/hahomematic

**Comment by Baxxy13** [comment]
Jens, woher weißt du denn das genau diese Integration genutzt wird und nicht die andere? 😉 

Um mal die Eingangsfrage zu beantworten:
Bei mir läuft alles, auch die erwähnten HM Aktoren.

**Reply by jens-maus** [reply]
Genau deshalb hab ich ja in meinem ersten Post davon geschrieben das ich kein Problem mit der "Homematic(IP) Local" Integration via HACS (https://github.com/jens-maus/RaspberryMatic/wiki/HomeAssistant-Integration) sehen kann. Und die ist IMHO inzwischen state-of-the-art und die alte sollte nicht mehr verwendet werden.

**Reply by Baxxy13** [reply]
Ja gut, trotzdem wissen wir nicht welche Integration @bratbecker  und/oder @colognebroker nutzen.

Ist es "Homematic(IP) Local" dann muss die Integration wohl einfach mal geupdatet werden.
Aktuell ist gerade 1.39.0

**Reply by jens-maus** [reply]
> Ist es "Homematic(IP) Local" dann muss die Integration wohl einfach mal geupdatet werden.
> 
> Aktuell ist gerade 1.39.0 

Stimmt natürlich, die Version der Integration ist auch wichtig!

**Reply by bratbecker** [reply]
Also ich setze wohl eine schon ältere Version ein, sagt jedenfalls die Diagnoseinfo.
Werde ich gleich mal updaten und schauen.

     ]
    },
    "homematicip_local": {
      "version": "1.37.3",
      "requirements": [
        "hahomematic==2023.6.0"

**Reply by bratbecker** [reply]
So, nach Update von HA und HM IP funktioniert nun alles wieder. 
Die HomeMatic IP Version 1.39.0 verlangt eine HA Version >=2023.7.0
Also habe ich erst HA auf die neueste Version gebracht (mit der HM IP 1.37.3 funktionieren dann, wie beschrieben, einige Sachen nicht) 
Anschließend die  HM IP über HACS neu installiert.

**Reply by jens-maus** [reply]
Ok, gut zu wissen das es mit der aktuellen 1.39.0 Version der Homematic(IP) Local Integration funktioniert. Die habe ich natürlich selbst schon seit Erscheinungsdatum am laufen.

Da bin ich einfach total naiv davon ausgegangen, das wenn jemand der das aktuellste HA installiert hat natürlich auch seine installierten HACS Integrationen immer aktuell hält 😜

**Reply by bratbecker** [reply]
Und ich bin als Newbie (vor 4 Wochen von FHEM auf HA umgestiegen) davon ausgegangen, dass sich die Integrationen irgendwie melden, wenn es was aktuelleres gibt. 😀 Muss wohl noch viel lesen🧐

**Reply by Baxxy13** [reply]
- entweder regelmäßig bei HACS reingucken...
- oder den HACS-Sensor auf ein Dashboard legen...
- oder eine Automatisierung anlegen, wenn HACS-Update dann alle Lampen auf Rot... :wink:

**Reply by fastbag** [reply]
Heute habe ich das gleiche Problem mit meinen Jalousie Aktoren. 
Diese Entität wird nicht mehr von der Integration homematicip_local bereitgestellt. Wenn die Entität nicht mehr verwendet wird, lösche sie in den Einstellungen.

**Reply by fastbag** [reply]
Nach einem Update und gestern nach einem Stromausfall sind im Addon wenn ich mich einlogge alle Jalousieschalter steuerbar, nur über HomeAssistant nicht mehr, alle auf undefined %

**Reply by fastbag** [reply]
Logger: homeassistant.config_entries
Quelle: config_entries.py:2757
Erstmals aufgetreten: 13:37:39 (1 Vorkommnisse)
Zuletzt protokolliert: 13:37:39

Error occurred loading flow for integration homematicip_local: No module named 'custom_components.homematicip_local.const'

**Reply by fastbag** [reply]
![image](https://github.com/jens-maus/RaspberryMatic/assets/46375317/d98d1b74-e478-4edf-90aa-560c350f64bc)

**Comment by lkdkknsckmsclmsc** [comment]
Hallo,
ich habe den Homeassistant heute als PVM in Proxmoxx eingerichtet, nun kann ich meine ganzen Homematic-Geräte nicht mehr über die Homematic-App ansprechen :-(
Bräuchte dringend Hilfe !

**Reply by lkdkknsckmsclmsc** [reply]
Sehe gerade, dass bei allen Geräten das Antennen-Symbol durchgestrichen ist...

**Reply by lkdkknsckmsclmsc** [reply]
OK, nach Neustart des Access Points geht zum Glück wieder alles :-)
